lib/routes/account.js

```diff
@@ -910,30 +910,35 @@ module.exports = function (
             return getGeoData(ip)
               .then(
                 function (geoData) {
-                  mailer.sendNewDeviceLoginNotification(
-                    emailRecord.email,
-                    {
-                      acceptLanguage: request.app.acceptLanguage,
-                      flowId: flowId,
-                      flowBeginTime: flowBeginTime,
-                      ip: ip,
-                      location: geoData.location,
-                      timeZone: geoData.timeZone,
-                      uaBrowser: sessionToken.uaBrowser,
-                      uaBrowserVersion: sessionToken.uaBrowserVersion,
-                      uaOS: sessionToken.uaOS,
-                      uaOSVersion: sessionToken.uaOSVersion,
-                      uaDeviceType: sessionToken.uaDeviceType
-                    }
-                  )
-                  .catch(e => {
-                    // If we couldn't email them, no big deal. Log
-                    // and pretend everything worked.
-                    log.trace({
-                      op: 'Account.login.sendNewDeviceLoginNotification.error',
-                      error: e
+                  db.accountEmails(sessionToken.uid)
```

Comment:
it seems we can improve the code here by doing `return  db.accountEmails(sessionToken.uid)` and continuing the `.then` chain from geoDb.

I also wonder if it is better to do the `db.accountEmails` first or do `getGeoData` first